### PR TITLE
V4

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+      - image: circleci/node:latest
+    working_directory: ~/repo
+
+jobs:
+  minor_release:
+    executor: node_sdk
+    steps:
+      - checkout
+      - build_and_release:
+          release_type: major
```
Comment (przywartya) [inline .circleci/config.yml]: job is  `minor_release` but `release_type` is `major`

File: .circleci/config.yml [hunk 2]
```diff
@@ -0,0 +1,103 @@
...
+          command: echo export MAJOR_VER=$SEMVER | cut -d. -f1 >> $BASH_ENV
+      - run:
+          name: Echo SEMVER
+          command: echo "Semver v${SEMVER}"
+      - run:
+          name: Get latest LSB release
+          command: sudo apt-get -y update && sudo apt-get -y install lsb-release
+      - run:
+          name: Install Google Cloud SDK
+          command: export CLOUD_SDK_REPO="cloud-sdk-$(lsb_release -c -s)" && echo "deb http://packages.cloud.google.com/apt $CLOUD_SDK_REPO main" | sudo tee -a /etc/apt/sources.list.d/google-cloud-sdk.list && curl https://packages.cloud.google.com/apt/doc/apt-key.gpg | sudo apt-key add - && sudo apt-get update -y && sudo apt-get install google-cloud-sdk -y
```
Comment (przywartya) [inline .circleci/config.yml]: Can we somehow wrap this line? It's hard to read this 😝

File: .circleci/config.yml [hunk 3]
```diff
@@ -0,0 +1,103 @@
...
+            gcloud auth activate-service-account --key-file=${HOME}/gcp-key.json
+      - run:
+          name: Install Auto-identify in lib
+          working_directory: ~/repo/lib
+          command: npm i git+https://${GITHUB_TOKEN}:x-oauth-basic@github.com/dreamdata-io/auto-identify.git#auto-identify-lib --no-save
+      - run:
+          name: Install rest lib dependencies
+          working_directory: ~/repo/lib
+          command: npm install
+      - run: cd lib && pwd && ls && cd node_modules && pwd && ls
```
Comment (przywartya) [inline .circleci/config.yml]: ` cd lib && pwd && ls && cd node_modules && pwd && ls` is it  necessary ? 😄

File: webpack.dev.js [hunk 4]
```diff
@@ -1,34 +1,40 @@
...
+          loader: 'babel-loader'
+        }
+      },
+      {
+        test: /\.js$/,
+        loader: 'string-replace-loader',
+        options: {
+          multiple: [
+            { search: 'api.dreamdata.cloud/v1', replace: 'localhost:8080/v1' },
+            { search: `'https://'`, replace: `'https://'` }
```
Comment (przywartya) [inline webpack.dev.js]: I love polish chocolate

File: .circleci/config.yml [hunk 5]
```diff
@@ -63,24 +63,24 @@ commands:
...
-      #     name: Copy script in CDN with 5min Cache
-      #     command: |
-      #       if [<<parameters.release_type>>== major]
-      #              gsutil -h "Cache-Control:public, max-age=300" -h "Content-Type:text/javascript; charset=utf-8" -h "Content-Encoding:gzip" \
-      #              cp  lib/index.js.gz ${GSUTIL_BUCKET_PATH}/<<parameters.account_slug>>/v${MAJOR_VER}/dreamdata.min.js
-      #       else
-      #              gsutil -h "Cache-Control:public, max-age=300" -h "Content-Type:text/javascript; charset=utf-8" -h "Content-Encoding:gzip" \
-      #              cp  lib/index.js.gz ${GSUTIL_BUCKET_PATH}/<<parameters.account_slug>>/v${MAJOR_VER}/${SEMVER}/dreamdata.min.js
-      #       fi
+      # - run: cd lib && pwd && ls && cd node_modules && pwd && ls
```
Comment (przywartya) [inline .circleci/config.yml]: OK!

File: .circleci/config.yml [hunk 6]
```diff
@@ -12,7 +17,7 @@ jobs:
     steps:
       - checkout
       - build_and_release:
-          release_type: major
+          release_type: minor
```
Comment (przywartya) [inline .circleci/config.yml:20]: OK!

**Review by przywartya — APPROVED** [review]
👍

File: webpack.dev.js (hunk 4 above)
Comment (ghost) [reply webpack.dev.js]: typo - supposed to replace to http, fixed in 4.0.1

File: .circleci/config.yml (hunk 2 above)
Comment (ghost) [reply .circleci/config.yml]: wrapped

File: lib/init-script/init.js [hunk 7]
```diff
@@ -1,79 +1,82 @@
...
-            var method = item.shift();
-            if (dreamdata[method]) dreamdata[method].apply(dreamdata, item);
-        }
-    });
-    dreamdata.page();
+  dreamdata.load(function () {
+    // On load init our integrations
+    let { initialize, plugins } = dreamdata;
+    initialize({ 'Dreamdata.io': { apiKey: '<API_KEY>', apiHost: '<api.subdomain.domain.io>' } });
+    plugins.autoIdentify();
```
Comment (ghost) [inline lib/init-script/init.js:72]: @przywartya here

**Pull request merged**